frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/status-card.tsx

```diff
@@ -74,6 +81,60 @@ export const StatusCard: React.FC<DashboardItemProps> = ({
     t,
   );
 
+  const healthChecksGetterOrPresent = (checkPresence: boolean = false) => {
+    const cephDetails = data?.[0]?.status?.ceph?.details;
+    const pattern = /[A-Z]+_*|error/g;
```

Comment:
This handles both warnings and alerts alike so I thought it was better to have a more general approach. Also, at a time, only one is active (Warning or Error).

Reply:
```
WARNING:
  <[A-Z]+_*>: <message> // a key in case of a warning (matching the first half of the regex
ERROR:
  error: <message> // a key in case of an error (always specified as "error" in the cephDetails object)
```

This regex handles both warnings and errors, in a general manner. Not sure if a specific use case for errors is needed when it can be achieved with a single regex?